packages/assets-controllers/src/NftController.ts

```diff
@@ -218,11 +218,11 @@ const controllerName = 'NftController';
 /**
  * The external actions available to the {@link NftController}.
  */
-export type AllowedActions =
+type AllowedActions =
   | AddApprovalRequest
   | NetworkControllerGetNetworkClientByIdAction;
 
-export type AllowedEvents =
+type AllowedEvents =
```

Comment:
We'll still want to export these so they can be used inside of the package. To exclude these from package-level exports, we need to update the `index.ts` file, specifically line 4: 

```diff
- export * from './NftController';
+ export {
+   NftControllerMessenger, 
+   NftControllerActions, 
+   NftControllerGetStateAction,
...
+ } from './NftController';
```

Reply:
I am not sure what's being used what's not we have a lot of exports within the controller we can clean this export later ?

Reply:
Everything that previously had an `export` keyword attached would have been package-level exports and should be added to the list. 

The new exports that were added (`NftControllerActions`, `NftControllerGetStateAction`, `NftControllerEvents`, `NftControllerStateChangeEvent`) or renamed (`getDefaultNftControllerState`) in this PR and are intended as package-level exports need to be listed in the changelog as well.

Reply:
@mcmire RE: https://github.com/MetaMask/core/pull/4342#issuecomment-2140812973

This following needs to be added to the changelog for this PR:

```md
### Added

- Adds and exports new types: `NftControllerActions`, `NftControllerGetStateAction`, `NftControllerEvents`, `NftControllerStateChangeEvent` ([#4310](https://github.com/MetaMask/core/pull/4310)).
```